server/app.ts

```diff
@@ -33,16 +33,36 @@ const version = Date.now().toString()
 const production = process.env.NODE_ENV === 'production'
 const testMode = process.env.NODE_ENV === 'test'
 const RedisStore = connectRedis(session)
+const metricsMiddleware = promBundle({
```

Comment:
any chance this can move out to a module? app.ts is way too big already (I have a wip branch to break it up more)

Reply:
Can try to... not done much in the way of modules but will give it a shot